charts/adguard/templates/primary/route.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if .Values.primary.route.enabled -}}
+apiVersion: {{ .Values.primary.route.apiVersion | default "gateway.networking.k8s.io/v1" }}
+kind: {{ .Values.primary.route.kind | default "HTTPRoute" }}
+metadata:
+  name: {{ template "common.fullname" . }}
+  annotations: {{ toYaml .Values.primary.route.annotations | nindent 4 }}
+  labels:
+    {{- include "common.labels" . | nindent 4 }}
```

Comment:
On Ingress we have these additional labels:

```
app.kubernetes.io/name: {{ template "common.name" . }}
app.kubernetes.io/instance: {{ .Release.Name }}
app.kubernetes.io/component: "adguard"
```

Is it intentional that they are missing here?